.github/ISSUE_TEMPLATE/015-browser-review.yaml

```diff
@@ -0,0 +1,89 @@
+name: Browser-driven specification review
+description: >
+  If your browser is considering shipping a feature
+  that hasn't had a Working Group take it through wide review,
+  please use this form.
+title: "Browser review: "
+labels:
+  - "Progress: untriaged"
+  - "Review type: later review"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for sending us your specification review. If a Working Group generally agrees about this feature, we prefer to have them take it through wide review. If they're willing to do that, please [switch to the WG new specification review template](https://github.com/w3ctag/design-reviews/issues/new?template=005-wg-new-spec-review.yaml). Otherwise, if you're planning to ship the feature anyway, please continue to file the review with this form.
```

Comment:
This still reads like, "hey TAG, review our proprietary feature". 

I don't think the TAG should spend time reviewing things that haven't initiated a standardization process (even if it's an incubation process).

Reply:
I'll add a note that this review process is only for features in a Community Group or similar incubation process, or later. But there's still a lot of space between entering a CG and having a WG ready to advance a feature to CR.